dev-python/protobuf-python/protobuf-python-3.19.4.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 2008-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/protocolbuffers/protobuf"
+	EGIT_SUBMODULES=()
```

Comment:
Why?

Reply:
Otherwise `git-r3.eclass` would download (into `${DISTDIR}/git3-src`) copies of unused repositories which are listed in `.gitmodules` file:
https://github.com/protocolbuffers/protobuf/blob/master/.gitmodules:
```
[submodule "third_party/benchmark"]
	path = third_party/benchmark
	url = https://github.com/google/benchmark.git
[submodule "third_party/googletest"]
	path = third_party/googletest
	url = https://github.com/google/googletest.git
	ignore = dirty
```
`dev-java/protobuf-java` and `dev-python/protobuf-python` would never use them, while C++ `dev-libs/protobuf` is patched to use system version of `dev-cpp/gtest` (and it does not use `dev-cpp/benchmark`).